### PR TITLE
The pop ups don't work

### LINKED ISSUE #78: The pop-ups dont work
Steps to reproduce the error:
* press `+1` button more than once(twice or thrice or ..) consecutively, without giving the pop-up time to fade away.
* Now, before it getting fade away, press the cross button on the pop-up.
* Now, if you again press the `+1` button any number of times, the pop-up for that wont function.

P.S: this is with all the four pop-ups, so the steps would be same, just replace the name of the button.

### PR DESCRIPTION
**Pull request by kri-bhawna**
fixes #78

### CODE REVIEW

**Comment by kri-bhawna** [comment]
@Raghwendra-Dey  I have removed the bug regarding pop-ups not working properly.
I have checked everything from my side. Do give your feedback regarding this.

**Comment by Raghwendra-Dey** [comment]
@kri-bhawna you should write fixes #78 . Not only the issue_number. Adding fixes, closes, resolved, etc. automatically closes that issue, if the PR is merged.

**Comment by Raghwendra-Dey** [comment]
And Also, I think you have not made your local clone apt with the upstream:master, and worked on that same old copy, and hence you patch don't have the recent changes. Please sync you path with the recent version of the master, and then make your changes and update the PR.
TIP: Never forget to sync you clone with the upstream:master, before starting on a completely new issue.
tL;dr make the site look exactly same as it looks now, and then remove the bug.

**Comment by kri-bhawna** [comment]
@Raghwendra-Dey  I checked for the files which need to be updated (index.html and style.css), and tried to update them. But, it is showing that branch has conflicts. I checked the code and was trying to resolve the conflicts. But both the codes which are having conflicts are the same. Please help me regarding this. Should I create a new PR or should I keep the master branch style.css in this PR only and resolve conflicts. Also, I think that creating a new PR will reduce the probability of any further mistakes.

**Comment by Raghwendra-Dey** [comment]
> @Raghwendra-Dey  I checked for the files which need to be updated (index.html and style.css), and tried to update them. But, it is showing that branch has conflicts. I checked the code and was trying to resolve the conflicts. But both the codes which are having conflicts are the same. Please help me regarding this. Should I create a new PR or should I keep the master branch style.css in this PR only and resolve conflicts. Also, I think that creating a new PR will reduce the probability of any further mistakes.

First of all make sure you make a backup of your current version of the repo(having all the changes and bug fixes) so that you dont lose some work.
Then you can have a look at [this](https://www.earthdatascience.org/courses/intro-to-earth-data-science/git-github/github-collaboration/update-github-repositories-with-changes-by-others/#:~:text=You%20can%20update%20your%20fork,be%20updated%20in%20your%20fork.) This would update your fork's master with the master of the original repo. Than you can try to again git clone in somewhere else(maybe if you were previously working in `Downloads`, so now try to git clone on `Desktop`) now just redo the changes you have done before on this new clons's files(you can refer your previous backup files for help) to fix that bug.

**Comment by kri-bhawna** [comment]
@Raghwendra-Dey  I tried  the method suggested by you, but it is not working. May I create another PR for this or use the resolve conflicts option given at the bottom of this tab and save master branch code (as I have not touched the style.css code)?

**Comment by Raghwendra-Dey** [comment]
Either of them is fine
